### PR TITLE
Fixing Walker Tree Items and Adding walker tools

### PR DESCRIPTION
**Pull request by kyleshrader**
* Added Tool Pod
* Added Sawblade Tool
* Added Long Sawblade
* Updated walker legs to be 1 of 2 since 1 of 4 was inaccurate
* Fixed many wing values that were incorrect from the wiki
* Renamed a durability upgrade to armor
* Added hornet walker wings
* Fixed typo of rupu pelt to rupu vine
* Fixed incorrect rope count on schmetterling body
* Added proxy license to proxy requirements
* Fixed incorrect wood shaft count on armored proxy legs
* Added missing fiber ingredient to titan body

### CODE REVIEW

**Comment by kyleshrader** [comment]
sawblade_tool
long_sawblade
tool_pod
proxy_walker
proxy_walker_legs_armored
schmetterling_walker
hornet_walker
titan_walker

File: src/itemObjGen.txt [hunk 1]
```diff
@@ -4035,13 +4078,13 @@
   crafting[]
     ingredients[]
       name = Reinforced Plank
```
Comment (RodEsp) [inline src/itemObjGen.txt:4080]: This should be `Wood Shaft` not Reinforced Plank.

File: src/itemObjGen.txt [hunk 2]
```diff
@@ -4061,13 +4104,13 @@
   crafting[]
     ingredients[]
       name = Wood Shaft
```
Comment (RodEsp) [inline src/itemObjGen.txt:4106]: This should be `Reinforced Plank` not Wood Shaft.

File: src/itemObjGen.txt (hunk 1 above)
Comment (kyleshrader) [reply src/itemObjGen.txt]: Great catch!

**Comment by kyleshrader** [comment]
Great catch on those two ingredients. Thanks for the detailed review.

The wings that I had initially were mostly data I got from the wiki. In fact, most of the issues I found were from the data I had gotten from them... Maybe we should contribute back to them to help out.

**Pull request merged**